modules/gitops-synchronizing-cluster-configs-application.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * configuring-an-openshift-cluster-with-argo-cd.adoc
+
+[id="synchronizing-cluster-configs-application_{context}"]
+= Synchronizing `cluster configs` application
+
+. In the Argo CD dashboard, notice that the `cluster-configs` Argo CD application has the statuses `Missing` and `OutOfSync`. Because the application was configured with a manual sync policy, Argo CD does not sync it automatically.
+
+. Click the *Sync* button on the `cluster-configs` tile, review the changes, and then click *Synchronize*.
```

Comment:
`cluster-configs` bold, remove back ticks

Reply:
Suggest adding this sentence here as additional information or as a note: Argo CD will detect any changes in the Git repository automatically. If the configurations are changed, Argo CD will change the status of the `cluster-configs` to `OutOfSync`. You can modify the synchronization policy for Argo CD to automatically apply changes from your Git repository to the cluster.